src/pybind/mgr/dashboard/controllers/health.py

```diff
@@ -305,3 +367,85 @@ def get_cluster_fsid(self):
     @Endpoint()
     def get_telemetry_status(self):
         return mgr.get_module_option_ex('telemetry', 'enabled', False)
+
+    @Endpoint()
+    @EndpointDoc("Get a quick overview of cluster health at a moment, analogous to the ceph status command in CLI.",
+                 responses={200: HEALTH_SNAPSHOT_SCHEMA})
+    def snapshot(self):
+        data = CephService.send_command('mon', 'status')
```

Comment:
my only concern here is this one right here. But not a big deal I suppose.

Reply:
Same here, I will test this out actually to see results that can back but if compared to minimal definitely improvement